### DISCUSSION TITLE
Limit one person per stream

### DISCUSSION THREAD

**Discussion by magebot1** [bluenviron/mediamtx#1773] [Questions]
Is possible to limit a maximum of 1 connection per stream?

**Comment by aler9** [comment]
Hello, there's a feature request here: #1286

**Reply by pepeu93** [reply]
Cool!
I'm also looking for this, if you implement this feature how would it be? 
It would kick the previous connection or block the new connection?

**Reply by aler9** [reply]
Of course it would block new connections, it would be senseless if done otherwise

[a comment by an automated account is not shown]
